src/components/forms/PlanForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,5 +1,171 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`containers/forms/PlanForm - Dynamic Form Activities removing dynamic activities works correctly: Changed activity conditions text input names 1`] = `Array []`;
```

Comment:
These snapshot tests codify the expected input field names.  This is important because if they are mismatched then the form will not work correctly.